ctapipe/atmosphere.py

```diff
@@ -131,7 +151,7 @@ def peek(self):
         return fig, axis
 
     @classmethod
-    def from_table(cls, table: Table):
+    def from_table(cls, table: QTable):
```

Comment:
I didn't use QTables here explicitly because they drop some metadata when writing. I haven't checked if that behavior is fixed in the latest astropy, however.  Due to that, I generally prefer to use bare Tables for internal variables, and only cast to a QTable when needed  for simplicity.

Reply:
So should I revert everything back to Tables?

Reply:
Maybe better, I noticed that the ecsv output of a QTable is much more complex and we use mostly `Table` everywhere else.

So if you want to use QTable internally because that saves a couple of `.quantity`, you could do `table = QTable(table, copy=False)` as first line.

Reply:
I've done exactly this now